Makefile

```diff
@@ -76,8 +97,9 @@ uninstall:
 	rm -f $(DESTDIR)$(MANDIR)/man1/maxcso.1
 
 clean:
-	rm -f $(SRC_CXX_OBJ) $(CLI_CXX_OBJ) $(ZOPFLI_C_OBJ) maxcso
-	$(MAKE) -C 7zip clean
+	rm -rf -- $(OBJDIR)
+	rm -f maxcso
+	$(MAKE) -C $(SRCDIR)/libdeflate clean
```

Comment:
It seems like this was changed from 7zip to libdeflate.  I could believe I missed adding libfdeflate, but don't we need both then?

Or just skipping because 7zip is going to end up in objdir too?  If so, I'd recommend still putting the make -C, just in case we add something to clean later.

-[Unknown]

Reply:
I think libdeflate was just missing from the `clean` target before so I added it.

I removed 7zip because the logic was that both the main Makefile and 7zip's `Makefile` will remove `$(OBJDIR)`, so if both clean targets ran at once it would be redundant.